e2e-tests/playwright/specs/functional/plugins/demo_plugin_installation.spec.ts

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {AdminConfig} from '@mattermost/types/config';
+
+import {test, expect} from '@mattermost/playwright-lib';
+
+test('should install and enable demo plugin from URL', async ({pw}) => {
+    const {adminClient} = await pw.initSetup();
+
+    // Enable public links before installing plugin
+    await adminClient.updateConfig(
+        pw.mergeWithOnPremServerConfig({
+            FileSettings: {EnablePublicLink: true},
+        } as Partial<AdminConfig>),
+    );
```

Comment:
Is there a `patchConfig` we can use instead? Ideally we don't have to care at the code-level about the "whole config" and just use the discrete endpoint instead.